docs-crawler/docs/aggregate_crawler_output.py

```diff
@@ -142,7 +142,7 @@ def hydrate_keys(input_dict, output, from_lang, to_lang):
         return output
 
     def output_data(self):
-        with open('../../static/mapped_commands_full.json', 'w',
+        with open('../../pythreepio/static/mapped_commands_full.json', 'w',
```

Comment:
any static files **must** be located in pythreepio in order for it to be included in the build.